### PR TITLE
Revised description of stats module

### PR DESCRIPTION
**Pull request by chrisb83**
Related discussion in #97

### CODE REVIEW

File: scipy-1.0/paper.tex [hunk 1]
```diff
@@ -920,27 +920,17 @@ \subsection*{Numerical optimization}
...
-is either a \texttt{numpy.random.RandomState} object or an integer used to
-generate such an object, to provide fine-grained user-level control over the
-random state of a workflow.  Currently about 80 continuous univariate
-distributions, 12 discrete univariate distributions and 10 multivariate
-distributions are implemented.  Key recent distributions added to
-\texttt{scipy.stats} include the histogram-based distribution in
-\texttt{scipy.stats.rv\_histogram} and the multinomial distribution in
-\texttt{scipy.stats.multinomial} (used, for example, in natural language
-processing\cite{Griffiths5228}). \texttt{multinomial} is the first and so far
-the only multivariate discrete distribution available in SciPy.
```
Comment (rgommers) [inline scipy-1.0/paper.tex]: I think the issue with dropping all of this is that the whole section is about recent improvements, and now there's only the generic description left. The description bit is nicer now, but I would suggest to at least keep `multinomial`.

Comment (chrisb83) [reply scipy-1.0/paper.tex]: agree, I added back that part

**Comment by WarrenWeckesser** [comment]
LGTM

File: scipy-1.0/subpackages.tex [hunk 2]
```diff
@@ -107,13 +107,16 @@
     combinatorial functions \texttt{comb} and \texttt{factorial};
     and more.
 \item[\texttt{stats}]
-    The \texttt{stats} subpackage provides a large collection of continuous
-    and discrete probability distributions, each with methods to compute
+    The \texttt{stats} subpackage provides: a large collection of continuous
```
Comment (mdhaber) [inline scipy-1.0/subpackages.tex]: Colon is not appropriate here according to Rule 2 (https://www.grammarbook.com/punctuation/colons.asp)

File: scipy-1.0/subpackages.tex [hunk 3]
```diff
@@ -107,13 +107,16 @@
...
+    and discrete \emph{probability distributions}, each with methods to compute
```
Comment (mdhaber) [inline scipy-1.0/subpackages.tex]: Does this need to be emphasized?

File: scipy-1.0/paper.tex [hunk 4]
```diff
@@ -920,27 +920,22 @@ \subsection*{Numerical optimization}
...
+The \texttt{scipy.stats} module contains more than 100 probability
+distributions (96 continuous and 13 dicrete univariate distributions
+and 10 multivariate distributions). The implementation relies on a
+consistent framework that provides methods to sample random variates,
+to evaluate the cumulative distribution function (cdf), the probability
+density function (pdf) and to fit parameters for every distribution.
```
Comment (mdhaber) [inline scipy-1.0/paper.tex]: The structure of this list "to sample random variates, to evaluate..." is:
[infinitive verb] [noun], [infinitive verb] [noun], [noun], and [infinitive verb] [noun].
Please consider:
[infinitive verb] [noun], [infinitive verb] [noun], [infinitive verb] [noun], and [infinitive verb] [noun].
or
[infinitive verb] [noun], [infinitive verb] [noun] and [noun], and [infinitive verb] [noun].

File: scipy-1.0/paper.tex [hunk 5]
```diff
@@ -920,27 +920,22 @@ \subsection*{Numerical optimization}
...
-random state of a workflow.  Currently about 80 continuous univariate
-distributions, 12 discrete univariate distributions and 10 multivariate
-distributions are implemented.  Key recent distributions added to
-\texttt{scipy.stats} include the histogram-based distribution in
-\texttt{scipy.stats.rv\_histogram} and the multinomial distribution in
-\texttt{scipy.stats.multinomial} (used, for example, in natural language
-processing\cite{Griffiths5228}). \texttt{multinomial} is the first and so far
-the only multivariate discrete distribution available in SciPy.
+The \texttt{scipy.stats} module contains more than 100 probability
+distributions (96 continuous and 13 dicrete univariate distributions
```
Comment (mdhaber) [inline scipy-1.0/paper.tex]: "discrete" typo
The use of parentheses in this section might catch [unwanted attention](https://thecopyfox.com/2012/11/writing-tip-do-you-overuse-parentheses/). 
Consider whether they are necessary here. Perhaps:
The \texttt{scipy.stats} module contains more than 100 probability distributions: 96 continuous univariate distributions, 13 discrete univariate distributions, and 10 multivariate distributions.
Or simply
The \texttt{scipy.stats} module contains 96 continuous univariate distributions, 13 discrete univariate distributions, and 10 multivariate distributions.

File: scipy-1.0/paper.tex [hunk 6]
```diff
@@ -920,27 +920,22 @@ \subsection*{Numerical optimization}
...
+and 10 multivariate distributions). The implementation relies on a
+consistent framework that provides methods to sample random variates,
+to evaluate the cumulative distribution function (cdf), the probability
+density function (pdf) and to fit parameters for every distribution. 
+Generally, the methods rely on specific implementations for each
+distribution if available (such as a closed form expression of the
```
Comment (mdhaber) [inline scipy-1.0/paper.tex]: Instead of (), 
", such as" would work.

File: scipy-1.0/subpackages.tex [hunk 7]
```diff
@@ -107,13 +107,16 @@
...
     the PDF or PMF, CDF, moments and other statistics, generation of random
     variates, and more;
-    statistical tests, including Pearson's correlation, Spearman's rank-order
-    correlations, Kendall's tau, chi-squared test and its generalization as the
-    Cressie-Read power divergence, contingency table tests including Fisher's
-    exact test and Mood's median test, and many more;
-    and assorted transformations and statistics of data.
+    statistical tests, including tests on equality of means / medians /
```
Comment (ilayn) [inline scipy-1.0/subpackages.tex]: you can remove the whitespace around forward slashes. TeX will adjust it anyways.

File: scipy-1.0/subpackages.tex [hunk 8]
```diff
@@ -107,13 +107,16 @@
...
+    variance (such as the t-test) and tests whether a sample is drawn from a
+    certain distribution (such as the Kolmogorov-Smirnov test);
+    measures of correlation, including Pearson's, Kendall's and Spearman's
```
Comment (ilayn) [inline scipy-1.0/subpackages.tex]: maybe include explicitly what these names are `Pearson's $r$, Kendall's $W$,  and Spearman's $\rho$ coefficients`. I will leave the oxford comma to your taste :)

**Comment by chrisb83** [comment]
thanks for the feedback, i incorporated the changes

File: scipy-1.0/paper.tex [hunk 9]
```diff
@@ -920,27 +920,22 @@ \subsection*{Numerical optimization}
...
-\texttt{scipy.stats.rv\_histogram} and the multinomial distribution in
-\texttt{scipy.stats.multinomial} (used, for example, in natural language
-processing\cite{Griffiths5228}). \texttt{multinomial} is the first and so far
-the only multivariate discrete distribution available in SciPy.
+The \texttt{scipy.stats} module contains more than 100 probability
+distributions: 96 continuous and 13 discrete univariate distributions
+and 10 multivariate distributions. The implementation relies on a
+consistent framework that provides methods to sample random variates,
+to evaluate the cumulative distribution function (cdf) and the probability
+density function (pdf) and to fit parameters for every distribution.
```
Comment (mdhaber) [inline scipy-1.0/paper.tex]: This is one of those places where commas before the "and" at the end of a list are particularly helpful.
[Here](https://www.verbicidemagazine.com/2011/09/20/strippers-jfk-and-stalin-illustrate-why-you-should-use-the-serial-comma/) is another example.
I'll merge and change it later, if that's OK.

File: scipy-1.0/paper.tex (hunk 6 above)
Comment (mdhaber) [reply scipy-1.0/paper.tex]: Technically needs a comma, I think, but most people won't miss it and it can be fixed later.

**Pull request merged**